common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -1949,7 +1961,27 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "the HTTP client."),
     HIVE_DRUID_HTTP_READ_TIMEOUT("hive.druid.http.read.timeout", "PT1M", "Read timeout period for the HTTP\n" +
         "client in ISO8601 format (for example P2W, P3M, PT1H30M, PT0.750S), default is period of 1 minute."),
-
+    HIVE_DRUID_BASE_PERSIST_DIRECTORY("hive.druid.basePersisitDirectory", "/tmp",
+            "local base persist directory used while indexing  data"
+    ),
+    DRUID_SEGMENT_DIRECTORY("hive.druid.storage.storageDirectory", "/druid/segments"
+            , "druid deep storage location."),
+    DRUID_METADATA_BASE("hive.druid.metadata.base", "druid", "Default prefix for metadata tables"),
+    DRUID_METADATA_DB_TYPE("hive.druid.metadata.db.type", "mysql",
```

Comment:
Can you limit the options here to 'mysql' and 'postgres' (as in HIVE_DRUID_INDEXING_GRANULARITY)?